core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -128,7 +144,12 @@ oppia.factory('AssetsBackendApiService', [
     };
 
     var _isCurrentlyBeingRequested = function(filename) {
-      return _filesCurrentlyBeingRequested.indexOf(filename) !== -1;
+      for (var idx = 0; idx < _filesCurrentlyBeingRequested.length; idx++) {
```

Comment:
Take a look at Array.prototype.some() -- might be useful.

Reply:
Done

Reply:
Not sure anything changed here, but this was optional, so nvm.